pkg/api/testing/fuzzer.go

```diff
@@ -172,7 +172,15 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			i := int(c.RandUint64() % uint64(v.NumField()))
 			v = v.Field(i).Addr()
 			// Use a new fuzzer which cannot populate nil to ensure one field will be set.
-			fuzz.New().NilChance(0).NumElements(1, 1).Fuzz(v.Interface())
+			f := fuzz.New().NilChance(0).NumElements(1, 1)
+			f.Funcs(
+				func(m *api.MetadataFile, c fuzz.Continue) {
```

Comment:
Add a note here that explains that all other volume sources are fuzzable by the default fuzzer.